### PR TITLE
Cleaned up and documented constants

### PR DESCRIPTION
**Pull request by SpenceSellers**
The constants are a mess, but are fun to play with. I made an effort to reorder and document them so that further changes can be made more easily.

### CODE REVIEW

File: EvolvioColor/Board.pde [hunk 1]
```diff
@@ -541,4 +553,4 @@ class Board {
   public void unselect() {
     selectedCreature = null;
   }
-}
+}
```
Comment (mbertheau) [inline EvolvioColor/Board.pde]: Missing newline at end of file

File: EvolvioColor/Creature.pde [hunk 2]
```diff
@@ -632,4 +638,4 @@ class Creature extends SoftBody {
     double visionTotalAngle = rotation+visionAngles[i];
     return py+visionDistances[i]*Math.sin(visionTotalAngle);
   }
-}
+}
```
Comment (mbertheau) [inline EvolvioColor/Creature.pde]: Missing newline at end of file

File: EvolvioColor/EvolvioColor.pde [hunk 3]
```diff
@@ -238,4 +238,4 @@ float toWorldYCoordinate(float x, float y) {
   float angle = atan2(y-w, x-w);
   float dist = dist(w, w, x, y);
   return cameraY+grossify(sin(angle-cameraR)*dist+w, BOARD_HEIGHT)/zoom;
-}
+}
```
Comment (mbertheau) [inline EvolvioColor/EvolvioColor.pde]: Missing newline at end of file

File: EvolvioColor/NameGenerator.pde [hunk 4]
```diff
@@ -85,4 +85,4 @@ class NameGenerator {
   public boolean isVowel(char a) {
     return (a == 'a' || a == 'e' || a == 'i' || a == 'o' || a == 'u' || a == 'y');
   }
-}
+}
```
Comment (mbertheau) [inline EvolvioColor/NameGenerator.pde]: Missing newline at end of file

**Comment by SirSKillz** [comment]
Closing as it is outdated and changes were never made.

**Pull request closed without merging**